bwscanner/configutil.py

```diff
@@ -0,0 +1,28 @@
+import os.path
+from shutil import copyfile
+from ConfigParser import SafeConfigParser
+
+from bwscanner.logger import log
+
+
+def read_config(cfg_path):
+    log.debug('reading config %s' % cfg_path)
+    if not config_exists(cfg_path):
+        copy_config(cfg_path)
+    parser = SafeConfigParser()
+    parser.read([cfg_path])
+    # FIXME: handle section names
+    section = 'default'
+    return dict(parser.items(section))
+
+
+def config_exists(cfg_path):
+    return os.path.isfile(cfg_path)
+
+
+def copy_config(cfg_path, cfg_default_path=None):
+    if cfg_default_path is None:
+        cfg_default_path = os.path.join(os.path.dirname(os.path.dirname(
+                os.path.abspath(__file__))), 'data', 'config.ini')
```

Comment:
For this sort of thing, it's sometimes nicer to use `pkg_resources` (so that it still works if you've packaged a wheel or zip). See http://peak.telecommunity.com/DevCenter/PythonEggs#accessing-package-resources -- and also this interacts with `setup.py`, detailed here https://packaging.python.org/tutorials/distributing-packages/#package-data

Reply:
Changed in https://github.com/TheTorProject/bwscanner/pull/76/commits/342d0880f25c765bcb74fbf41d06cdad17f6f0db